blocks/api/raw-handling/phrasing-content-reducer.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * WordPress dependencies
+ */
+import { unwrap, replaceTag } from '@wordpress/utils';
+
+/**
+ * Internal dependencies
+ */
+import { isPhrasingContent } from './utils';
+
+/**
+ * Browser dependencies
+ */
+const { ELEMENT_NODE } = window.Node;
+
+function isBlockContent( node, schema = {} ) {
+	return schema.hasOwnProperty( node.nodeName.toLowerCase() );
+}
+
+export default function( node, doc, schema ) {
+	if ( node.nodeType !== ELEMENT_NODE ) {
+		return;
+	}
+
+	if ( node.nodeName === 'SPAN' ) {
+		const { fontWeight, fontStyle } = node.style;
+
+		if ( fontWeight === 'bold' || fontWeight === '700' ) {
+			node = replaceTag( node, 'strong', doc );
```

Comment:
There's a semantic meaning to `strong` which I don't think could be implied from styling of `font-weight: bold;` alone.

Semi-related: `<b>` is no longer "deprecated", and exists for the purposes of reflecting this distinction: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/b#Usage_notes

Same goes for `font-style: italic;` and `em` vs. `i`.

Reply:
Sure. But how else should we sanitise paste from e.g. Google Docs etc. This has also not been changed by this PR.

Reply:
I guess this is a behaviour that can change between paste and converting into blocks. In a next PR I will look into this, along with preserving other attributes that we'd like to preserve on conversion but not on paste.